### PR TITLE
no rdf support

### PR DESCRIPTION
**Pull request by whitewum**

### CODE REVIEW

**Review by randomJoe211 — DISMISSED** [review]

**Review by cooper-lzy — DISMISSED** [review]

File: docs-2.0/15.contribution/how-to-contribute.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+# How to Contribute
+
+## Before you get started
+
+### File an issue in the github or forum
```
Comment (randomJoe211) [inline docs-2.0/15.contribution/how-to-contribute.md:5]: on GitHub

File: docs-2.0/15.contribution/how-to-contribute.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+You are welcome to contribute any code or files to the project. But first we suggest you to raise an issue in the [github](https://github.com/vesoft-inc/nebula-graph) or in the [forum](https://discuss.nebula-graph.io/)
```
Comment (randomJoe211) [inline docs-2.0/15.contribution/how-to-contribute.md:7]: we suggest that you raise an issue, delete to

File: docs-2.0/15.contribution/how-to-contribute.md (hunk 1 above)
Comment (randomJoe211) [reply docs-2.0/15.contribution/how-to-contribute.md:5]: Check through the topic for GitHub

File: docs-2.0/15.contribution/how-to-contribute.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+ to start a discussion with the community. The community may have some previous discussion about the topic.
```
Comment (randomJoe211) [inline docs-2.0/15.contribution/how-to-contribute.md:8]: some previous discussions

File: docs-2.0/15.contribution/how-to-contribute.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+Here is the [vesoft inc. Contributor License Agreement](https://cla-assistant.io/vesoft-inc/).
+
+Click the **Sign in with Github to agree** button to sign the CLA.
+
+If you have any question, send an email to `info@vesoft.com`.
+
+## Step 1: Fork in the github.com
+
+The Nebula Graph project have many [repositories](https://github.com/vesoft-inc). Take [the graph engine repository](https://github.com/vesoft-inc/nebula-graph) for example:
```
Comment (randomJoe211) [inline docs-2.0/15.contribution/how-to-contribute.md:22]: The Nebula Graph project has

File: docs-2.0/15.contribution/how-to-contribute.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+
+1. Visit https://github.com/vesoft-inc/nebula-graph
+1. Click `Fork` button (top right) to establish an online fork.
```
Comment (randomJoe211) [inline docs-2.0/15.contribution/how-to-contribute.md:25]: Click the `Fork` button

File: docs-2.0/2.quick-start/0.FAQ.md [hunk 6]
```diff
@@ -72,6 +72,18 @@ The keywords in the preceding example are described as follows:
 
 Welcome to [add more practical scenarios](https://docs.nebula-graph.io/1.1/manual-EN/4.contributions/how-to-contribute/) and become a Nebula Graph contributor.
 
+## About Data Model
+
+## Does Nebula Graph support W3C `RDF` (or `SPARQL`, `GraphQL`)?
```
Comment (cooper-lzy) [inline docs-2.0/2.quick-start/0.FAQ.md:77]: `###`

File: docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md [hunk 7]
```diff
@@ -8,12 +8,14 @@ nGQL is a declarative graph query language for Nebula Graph. It allows expressiv
...
+- Supports graph traverse
 - Supports pattern match
 - Supports aggregation
 - Supports graph mutation
-<!-- - Supports distributed transaction (future release) -->
-- Supports pipe
+- Supports access control
+- Supports composite queries
+- Supports index
+- Supports most OpenCypher 9 graph query syntax (but mutations and controls syntax are not supported).
```
Comment (cooper-lzy) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md:18]: openCypher

**Review by cooper-lzy — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**